pkg/apiserver/experiment/experiment.go

```diff
@@ -342,6 +342,10 @@ func (s *Service) createStressChaos(exp *core.ExperimentInfo) error {
 		chaos.Spec.Duration = &exp.Scheduler.Duration
 	}
 
+	if exp.Target.StressChaos.ContainerName != nil {
```

Comment:
Do we need to check if `exp.Target.StressChaos` is `nil`?

Reply:
Seems all `Create Experiment API`s should need to be checked since all `ChaosInfo`s are pointers. 😢

We can plan a `refactor` for our server APIs.